azure-pipelines.yml

```diff
@@ -37,8 +37,22 @@ jobs:
         - AndroidEasingFunctions
         - Square
         - JakeWharton.Picasso2OkHttp3Downloader
+      preBuildSteps:
+        - pwsh: |
+            dotnet tool uninstall --global Cake.Tool
+            dotnet tool install --global Cake.Tool
+            dotnet tool install --global boots
+            boots https://aka.ms/xamarin-android-commercial-d16-8-macos
+          condition: eq(variables['System.JobName'], 'macos')
+        - pwsh: |
+            dotnet tool uninstall --global Cake.Tool
+            dotnet tool install --global Cake.Tool
+            dotnet tool install --global boots
+            boots https://aka.ms/xamarin-android-commercial-d16-8-windows
+          condition: eq(variables['System.JobName'], 'windows')
       tools:
-        - 'Xamarin.AndroidBinderator.Tool': '0.4.2'
+        - 'xamarin.androidbinderator.tool': '0.4.2'
+        - 'xamarin.androidx.migration.tool': '1.0.7.1'
```

Comment:
Add the boots tool here rather, then you can just have a prebuild step of using it. This will then make sure it installs properly on all OSes